src/trove_classifiers/__init__.py

```diff
@@ -1,5 +1,7 @@
+from natsort import natsorted
```

Comment:
This makes this a dependency of this package, which can make bootstrapping difficult. Is there any need to directly depend on this, given that we keep a list of sorted_classifiers anyway?

Reply:
I agree w/ @pradyunsg here.

Reply:
Sure, so drop the "public function to sort a list of classifiers" from https://github.com/pypa/trove-classifiers/issues/56, aka `classifiers_sorted`?

Reply:
Yup.